components/form/formCheckbox.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "preact/hooks";
+
+export type FormCheckboxType = {
+  value: string;
+  display: string;
+  path: string;
+  defaultChecked: boolean;
+};
+
+export const FormCheckbox = ({
+  value,
+  display,
+  path,
+  defaultChecked,
+}: FormCheckboxType) => {
+  const [checked, setChecked] = useState(defaultChecked);
```

Comment:
Our options are arrays of integers that we display as checkboxes, so we control the check state locally instead of in the greater `data` object so we don't have to handle adding/remove array values generically.